pkg/webhooks/server.go

```diff
@@ -581,7 +581,7 @@ func (ws *WebhookServer) excludeKyvernoResources(request *v1beta1.AdmissionReque
 				}
 				if !isAuthorized {
 					// convert RAW to unstructured
-					return fmt.Errorf("Resource is managed by Kyverno, can't be changed manually. You can edit generate policy to update this resource")
+					return fmt.Errorf("Resource is managed by a Kyverno policy and cannot be update manually. You can edit the policy %s to update this resource.",resource.GetName())
```

Comment:
This is not the policy name `resource.GetName()`, can we get the policy name somehow?

Reply:
There is no way...easies way it to add it in label

Reply:
Let's remove the policy name then.